### ISSUE TITLE
Performance issue

### ISSUE THREAD

**Issue by lxtGH** [Scalsol/mega.pytorch#76]
Hi! Thanks for opensource such project for the public.
I want to re-produce the mega-model results.
The performance is about 1 map lower than the paper. Has anyone successfully reproduced the experiment results ?

Mega-R101: 82.0  Paper: 82.9
Mega-R50: 76.1  Paper: 77.3

The reuslt is best results from multiple experiments ?

**Comment by Scalsol** [comment]
Hi, what config did you use? If you use 8 gpus, the result of ResNet101 backbone should be close to 82.9. The results of 4 gpus may be lower. The result of ResNet50 backbone is not reported in paper and its performance is not stable, all models with ResNet 50 backbone suffer from the same issue (we mention this in the note). We report this result just for reference :)

**Comment by lxtGH** [comment]
Mega-R101 uses 4 gpus(82.0, default config) when i use 8 gpus training, the reuslt is 81.5.

**Comment by lxtGH** [comment]
I will re-run default mega-r101-4gpus

**Comment by Scalsol** [comment]
I could not figure out why and more training iteration could not handle the problem. I haven't run many experiments with 8 gpus for at that time I have no enough GPU resources.

**Comment by lxtGH** [comment]
Report New results 82.4.  Is this caused by initial seed problems?

**Comment by Scalsol** [comment]
Yes, the initial seed will influence the overall result (In maskrcnn_benchmark it is not fixed, unlike mmdetection). In my experiments the fluctuation is around 0.3 AP. As AP50 is not a so stable metric compared to mAP, I think this is an acceptable result. If you have further question, you could find me out through WeChat: Scalsol

**Comment by lxtGH** [comment]
Thanks for your reply!
